src/UI/templates/default/ViewControl/viewcontrol.less

```diff
@@ -0,0 +1,20 @@
+.il-viewcontrol-section .glyphicon:before {
+  background-color: lighten(@brand-primary, 15%);
```

Comment:
lighten(@brand-primary, 15%) must move into a less variable. see: https://github.com/ILIAS-eLearning/ILIAS/tree/trunk/templates#colors. However, you might just use the bootstrap button-mixin for this: https://github.com/ILIAS-eLearning/ILIAS/blob/release_5-2/Services/UICore/lib/bootstrap-3.2.0/less/mixins/buttons.less. Probably use the same variables as are used for the default button:

```
.btn-default {
  .button-variant(@il-btn-standard-color; @il-btn-standard-bg; @il-btn-standard-border);
}
```